tron/serialize/runstate/dynamodb_state_store.py

```diff
@@ -47,38 +50,50 @@ def restore(self, keys) -> dict:
         vals = self._merge_items(first_items, remaining_items)
         return vals
 
-    def _get_items(self, keys: list) -> object:
-        items = []
+    def chunk_keys(self, keys: list) -> list:
```

Comment:
```suggestion
    def chunk_keys(self, keys: list[str]) -> list[list[str]]:
```

(i think)

Reply:
it's a list of lists of dicts according to the stack value, will update it with that